developers/articles/status-service.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Status Service
+redirect_from:
+  - /Developers/Articles/StatusService/
+  - /StatusService/
```

Comment:
those redirects are for keeping the links from the old MediaWiki website working, you don't really need to add them on a new page